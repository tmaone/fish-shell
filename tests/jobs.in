sleep 5 &
sleep 5 &
jobs -c
bg -23 1
fg 3
bg 3
sleep 1 &
disown
jobs -c
disown foo
disown (jobs -p)
or exit 0
