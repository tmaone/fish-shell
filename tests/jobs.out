Command
sleep
sleep
Command
sleep
sleep
